docs/en/observability/whats-new.asciidoc

```diff
@@ -3,21 +3,263 @@
 
 // tag::whats-new[]
+[discrete]
+== Onboard logs and metrics for Azure services in a few clicks with expanded Microsoft integration (technical preview)
+
+We've enhanced support for Azure monitoring use cases with our new expanded native Microsoft integration,
+letting users easily onboard logs and metrics for their Azure services to Elastic Observability in a few clicks.
+This includes subscription level data like activity logs, and all non-compute services, like Azure SQL Database and Azure Data Factory.
+
+Tag-based filters allow you to easily configure your setup to limit data collection to only specific resources.
+Logs and metrics are automatically written in the Elastic Common Schema (ECS) format,
+making it easy to correlate data from your Azure infrastructure and data from non-Azure sources.
+
+Finally, we're introducing the Elastic VM extension for Azure VMs and VM scale sets,
+allowing you to instrument your compute environment with {agent} in seconds.
+All installed extensions are automatically enrolled in Fleet,
+giving you the ability to manage a fleet of active Elastic VM extensions from {kib}.
+
+[role="screenshot"]
+image::images/7.13-azure-logs-metrics.png[Onboard logs and metrics for Azure services]
+
+[discrete]
+== Fleet Server provides better scalability and security (beta)
+
+Centrally manage a fleet of hundreds of thousands of {agent}s with {fleet-guide}/fleet-server.html[{fleet-server}],
+which releases in 7.13 in beta. {agent}s are installed on servers, corporate laptops, IOT, and more.
+{fleet-server} provides visibility into the status of {agent}s, makes policy updates,
+and executes fast response actions to security incidents. It's a new infrastructure component of the Elastic Stack.
+Previously, an API in {kib} provided central management in Fleet.
+We separated this component into an independent server to make it more scalable and secure.
+
+Scalability is critical because we have users with hundreds of thousands of agents.
+You can now enroll several times more agents into each node of {fleet-server} than you could previously with {kib}.
+It's designed to have a low resource footprint both for CPU and memory,
+and it can be scaled horizontally to handle the loads driven by our largest users.
+
+Speed is especially critical when it comes to executing response actions for security incidents.
+{agent}s maintain long polling connections to {fleet-server}. When an update or response action is applied,
+{fleet-server} can update the agents in seconds by responding to those requests.
+This provides a faster response time than you'd get with the default settings in Puppet, for example, which has a 30-minute run interval.
+
+{fleet-server} also allows us to enable deployment scenarios with stronger security, particularly in segmented networks.
+It's a common practice to use private networks with firewalls to reduce the surface area for attacks.
+{agent}s running on endpoints no longer need direct access to {kibana}.
```

Comment:
@EamonnTP @bmorelli25 this is rendering incorrectly. Should it say {kib}?

Reply:
Yes, it should. It looks like this was fixed in https://github.com/elastic/observability-docs/pull/709.